static/app/components/replays/breadcrumbs/timelineGaps.tsx

```diff
@@ -36,6 +38,11 @@ export default function TimelineGaps({durationMs, startTimestampMs, videoEvents}
     });
   }
 
+  trackAnalytics('replay.number_of_timeline_gaps', {
```

Comment:
Can maybe wrap with a `if (ranges) {` ?

Reply:
we want to know if it has 0 gaps too just to see the distribution!